src/main.ts

```diff
@@ -73,6 +77,18 @@ export function main(command?: string) {
         },
         outfit: { familiar: $familiar`Reagnimated Gnome` },
       },
+      {
+        name: "Beaten Up!",
+        do: () => {
+          throw "You got beaten up, which is bad";
+        },
+        completed: () => !have($effect`Beaten Up`),
```

Comment:
Do we automatically remove beaten up from the cleaver NC? Also I know some users (Hax) have lots of personal massagers which remove the effect so we should probably closet those. More opinions here would be useful: https://kolmafia.us/threads/track-combat-loss-when-adventuring.28096/

Reply:
It doesn't support cleaver NC yet